.env.example

```
@@ -40,6 +40,16 @@ ANOMALIES_RETENTION_DAYS=7
 # Web snapshot route: how often the server re-reads Supabase (edge-cached for all
 # viewers). Drives Supabase egress — see docs/cost-guardrails.md. 30-60s recommended.
 SNAPSHOT_REFRESH_SECONDS=30
+SUPABASE_SNAPSHOT_PAYLOAD_KB=35
+
+# Optional planning inputs for `npm run ops:supabase-free-tier`
+SUPABASE_EXPECTED_DB_SIZE_MB=120
+SUPABASE_EXPECTED_MONTHLY_ACTIVE_USERS=1000
+SUPABASE_EXPECTED_MONTHLY_THIRD_PARTY_USERS=0
+SUPABASE_EXPECTED_EDGE_FUNCTION_INVOCATIONS=0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_EXPECTED_EDGE_FUNCTION_INVOCATIONS key should go before the SUPABASE_EXPECTED_MONTHLY_ACTIVE_USERS key

(UnorderedKey)